projects/core/src/lib/download-layer/download-layer.component.ts

```diff
@@ -132,6 +135,24 @@ export class DownloadLayerComponent implements OnInit, OnDestroy {
         this.viewportSizeSubscription?.unsubscribe();
     }
 
+    unsubscribeTime(): void {
+        this.projectTimeSubscription?.unsubscribe();
+    }
+
+    setEditedExtent(): void {
+        this.editedExtent = true;
+        this.maybeUnsubscribeViewportSize();
+    }
+
+    setEditedResolution(): void {
+        this.editedResolution = true;
```

Comment:
this does not work for me. If I edit the resolution and zoom in, the value changes.

Reply:
Weird, pretty sure it worked before, but I can reproduce it. It now listens to the 'input' event instead of 'change' which should trigger every time the value is changed by the user.
https://github.com/geo-engine/geoengine-ui/pull/734/commits/3023343c6568b80dcc11724c531865fdaf8f6077